{
	"deprecated": false,
	"cpeName": "cpe:2.3:a:python:setuptools:58.3.0:*:*:*:*:*:*:*",
	"cpeNameId": "18B78665-EAA7-469F-9127-CDCA9029DFE3",
	"created": "2022-12-29T14:39:02.883",
	"lastModified": "2022-12-29T14:43:10.977",
	"titles": [
		{
			"title": "Python setuptools 58.3.0",
			"lang": "en"
		}
	],
	"refs": [
		{
			"ref": "https://pypi.org/project/setuptools/#history",
			"type": "Version"
		},
		{
			"ref": "https://pyup.io/posts/pyup-discovers-redos-vulnerabilities-in-top-python-packages/",
			"type": "Advisory"
		}
	]
}
